examples/OpenGL45/arrayReductionByMin_version1.shadertrap

```diff
@@ -0,0 +1,55 @@
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GL 4.5
+
+CREATE_BUFFER input_array SIZE_BYTES 1024 INIT_VALUES uint 110 51 11 69 209 142 17 109 164 101 115 39 108 2 75 56 59 166 153 116 143 86 57 133 169 192 27 136 104 213 30 96 13 68 252 114 71 102 201 139 66 227 226 79 89 176 106 0 152 230 182 165 163 43 16 202 49 105 149 132 171 249 141 237 53 228 8 33 168 15 25 177 196 31 82 212 247 231 255 117 23 122 77 19 126 44 120 131 194 1 238 95 4 218 157 5 225 193 155 179 7 232 180 248 140 251 188 10 223 113 92 48 61 127 190 67 40 124 63 243 158 159 74 54 151 191 200 203 199 24 3 125 246 204 187 50 62 146 244 170 118 205 181 154 46 12 99 236 240 147 233 172 134 32 210 161 156 178 221 78 85 135 145 52 211 28 18 41 37 14 22 29 34 185 250 36 81 253 195 144 64 235 88 242 217 220 121 94 239 20 207 123 150 65 97 208 70 84 186 6 224 9 60 175 162 128 129 87 137 234 91 26 241 58 245 138 72 45 21 42 173 93 38 83 160 198 103 80 76 73 167 107 215 219 119 35 197 184 222 174 254 214 112 206 98 47 216 111 148 183 229 100 130 55 189 90
+
+CREATE_BUFFER input_array_origin SIZE_BYTES 1024 INIT_VALUES uint 110 51 11 69 209 142 17 109 164 101 115 39 108 2 75 56 59 166 153 116 143 86 57 133 169 192 27 136 104 213 30 96 13 68 252 114 71 102 201 139 66 227 226 79 89 176 106 0 152 230 182 165 163 43 16 202 49 105 149 132 171 249 141 237 53 228 8 33 168 15 25 177 196 31 82 212 247 231 255 117 23 122 77 19 126 44 120 131 194 1 238 95 4 218 157 5 225 193 155 179 7 232 180 248 140 251 188 10 223 113 92 48 61 127 190 67 40 124 63 243 158 159 74 54 151 191 200 203 199 24 3 125 246 204 187 50 62 146 244 170 118 205 181 154 46 12 99 236 240 147 233 172 134 32 210 161 156 178 221 78 85 135 145 52 211 28 18 41 37 14 22 29 34 185 250 36 81 253 195 144 64 235 88 242 217 220 121 94 239 20 207 123 150 65 97 208 70 84 186 6 224 9 60 175 162 128 129 87 137 234 91 26 241 58 245 138 72 45 21 42 173 93 38 83 160 198 103 80 76 73 167 107 215 219 119 35 197 184 222 174 254 214 112 206 98 47 216 111 148 183 229 100 130 55 189 90
+
+CREATE_BUFFER result SIZE_BYTES 4 INIT_VALUES uint 255
+CREATE_BUFFER expected SIZE_BYTES 4 INIT_VALUES uint 0
+
+BIND_SHADER_STORAGE_BUFFER BUFFER input_array BINDING 0
+BIND_SHADER_STORAGE_BUFFER BUFFER result BINDING 1
+
+DECLARE_SHADER shader KIND COMPUTE
+#version 450
+
+layout(local_size_x=1, local_size_y=1, local_size_z=1) in;
+
+layout(std430, binding = 0) buffer input_array {
+  uint[256] data;
+};
+
+layout(std430, binding = 1) buffer result {
+  uint res;
+};
+
+void main() {
+	for(int i = 0; i < 256; ++i) {
+		res = min(res, data[i]);
+	}
+}
```

Comment:
As per my review of the other PR, please check this PR carefully and use consistent indentation.

Reply:
Done.